BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.java

```diff
@@ -48,7 +48,7 @@ private static BraintreeClientParams createDefaultParams(Context context, String
      * Create a new instance of {@link BraintreeClient} using a tokenization key or client token.
      *
      * @param context       Android Context
-     * @param authorization The tokenization key or client token to use. If an invalid authorization is provided, a {@link BraintreeException} will be returned in the {@link HttpResponseCallback}.
+     * @param authorization The tokenization key or client token to use. If an invalid authorization is provided, a {@link BraintreeException} will be returned in the {@link HTTPResponseCallback}.
```

Comment:
It's a bit confusing to me that we're referencing `HTTPResponseCallback` in this doc string. That's not a public interface, is it?

Reply:
Done!